docs/content.zh/release-notes/flink-2.2.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Release Notes - Flink 2.2"
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release notes - Flink 2.2
+
+These release notes discuss important aspects, such as configuration, behavior or dependencies,
+that changed between Flink 2.1 and Flink 2.2. Please read these notes carefully if you are
+planning to upgrade your Flink version to 2.2.
+
+### Table SQL / API
+
+#### Support VECTOR_SEARCH in Flink SQL
+
+##### [FLINK-38422](https://issues.apache.org/jira/browse/FLINK-38422)
+
+Apache Flink has initially integrated Large Language Model (LLM) capabilities, enabling semantic
+understanding and real-time processing of streaming data pipelines. This integration has been
+technically validated in scenarios such as log classification and real-time question-answering
+systems. However, the current architecture allows Flink to only use embedding models to convert
+unstructured data (e.g., text, images) into high-dimensional vector features, which are then
+persisted to downstream storage systems. It lacks real-time online querying and similarity analysis
+capabilities for vector spaces. The VECTOR_SEARCH function is provided in Flink 2.2 to enable users
+to perform streaming vector similarity searches and real-time context retrieval
+(e.g., Retrieval-Augmented Generation, RAG) directly within Flink.
+
+See more details about the capabilities and usages of
+Flink's [Vector Search](https://nightlies.apache.org/flink/flink-docs-release-2.2/docs/dev/table/sql/queries/vector-search/).
+
+#### Realtime AI Function
+
+##### [FLINK-38104](https://issues.apache.org/jira/browse/FLINK-38104)
+
+Flink already expanded the `ML_PREDICT` table-valued function (TVF) to perform
```

Comment:
Was the "already expanded " in 2.1? This implies there was a datastream version of  ML_PREDICT at 2.0 that was enhanced at 2.1. 

Maybe we should include SQL as well as the table api